reference/5.1/Microsoft.PowerShell.Core/About/about_Providers.md

```diff
@@ -1,51 +1,48 @@
-
-The data that a provider exposes appears in a drive, and you access the data
-in a path like you would on a hard disk drive. You can use any of the built-in
-cmdlets that the provider supports to manage the data in the provider drive.
-And, you can use custom cmdlets that are designed especially for the data.
+PowerShell providers are .NET programs that provide access to specialized data
+stores for easier viewing and management. The data appears in a drive, and you
+access the data in a path like you would on a hard disk drive. You can use any
+of the built-in cmdlets that the provider supports to manage the data in the
+provider drive. And, you can use custom cmdlets that are designed especially
+for the data.
 
 The providers can also add dynamic parameters to the built-in cmdlets. These
-are parameters that are available only when you use the cmdlet with the
-provider data.
+parameters are only available when you use the cmdlet with the provider data.
 
 ## Built-in providers
 
 PowerShell includes a set of built-in providers that you can use to
 access the different types of data stores.
 
-|Provider    |Drive        |Data store                                 |
-|------------|-------------|-------------------------------------------|
-|Alias       |Alias:       |PowerShell aliases                 |
-|Certificate |Cert:        |x509 certificates for digital signatures   |
-|Environment |Env:         |Windows environment variables              |
-|FileSystem  |(*)          |File system drives, directories, and files |
-|Function    |Function:    |PowerShell functions               |
-|Registry    |HKLM:, HKCU: |Windows registry                           |
-|Variable    |Variable:    |PowerShell variables               |
-|WSMan       |WSMan:       |WS-Management configuration information    |
+| Provider   |   Drive(s)  |OutputType                                                    |
+|----------- |------------ |--------------------------------------------------------------|
+|Alias       |Alias:       |System.Management.Automation.AliasInfo                        |
+|Certificate |Cert:        |Microsoft.PowerShell.Commands.X509StoreLocation               |
+|            |             |System.Security.Cryptography.X509Certificates.X509Certificate2|
+|Environment |Env:         |System.Collections.DictionaryEntry                            |
+|FileSystem  |C: (*)       |System.IO.FileInfo                                            |
+|            |             |System.IO.DirectoryInfo                                       |
+|Function    |Function:    |System.Management.Automation.FunctionInfo                     |
+|Registry    |HKLM: HKCU:  |Microsoft.Win32.RegistryKey                                   |
+|Variable    |Variable:    |System.Management.Automation.PSVariable                       |
+|WSMan       |WSMan:       |Microsoft.WSMan.Management.WSManConfigContainerElement        |
 
 (*) The FileSystem drives vary on each system.
```

Comment:
These are pre-existing, but that asterisk should be codefenced.

Reply:
the asterisk here is signifying a footnote to the table, not a syntactic element.

Reply:
I meant escaped, not codefenced.  `\*`.